thunder/tests/test_randomness.py

```diff
@@ -20,7 +20,7 @@ def test_uniform_philox(executor, device: str, dtype: dtypes.dtype):
     def func(shape, dtype, device, rng_seed, rng_offset):
         return ltorch.uniform_philox(shape, device=device, dtype=dtype, seed=rng_seed, offset=rng_offset)
 
-    cf = lc_compile(func, disable_preprocessing=True, executors_list=executor.executors_list())
+    cf = thunder.jit(func, executors_list=executor.executors_list())
```

Comment:
```suggestion
    cf = thunder.jit(func, executors=executor.executors_list())
```

`executors_list` is only supported for compat.